src/main.py

```diff
@@ -231,7 +231,7 @@ def run(self):
                                     sub.unsticky()
 
                                 print "Submitting game thread..."
-                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game"))
+                                sub = r.submit(self.SUBREDDIT, title, edit.generate_code(d,"game",self.TEAM_CODE))

```

Comment:
The codes I use come from the home_file_code and away_file_code properties. They also match the ones from the javascript you can find in mlb.com's source (copied and pasted for convenience: https://pastebin.com/raw/jubnhmAh)